### ISSUE TITLE
Homebridge not starting anymore after update

### ISSUE THREAD

**Issue by raymondsteen** [oznu/homebridge-config-ui-x#532]
**Describe The Bug:**
homebridge is not starting anymore after update
**To Reproduce:**
restart, restart docker, rebuild docker with latest

**Expected behavior:**

**Logs:**
[2/7/2020, 1:39:59 PM] [Config] [HapClient] Cannot load accessories. No Homebridge instances have been discovered.
[2/7/2020, 1:39:59 PM] [Config] [HapClient] Cannot load accessories. No Homebridge instances have been discovered.
[2/7/2020, 1:40:00 PM] Loaded config.json with 0 accessories and 2 platforms.
[2/7/2020, 1:40:00 PM] ---
[2/7/2020, 1:40:00 PM] Loaded plugin: homebridge-dummy
[2/7/2020, 1:40:00 PM] Registering accessory 'homebridge-dummy.DummySwitch'
[2/7/2020, 1:40:00 PM] ---
[2/7/2020, 1:40:00 PM] [Config] Starting terminal session
[2/7/2020, 1:40:00 PM] Loaded plugin: homebridge-edomoticz
[2/7/2020, 1:40:00 PM] Registering platform 'homebridge-edomoticz.eDomoticz'
[2/7/2020, 1:40:00 PM] ---
[2/7/2020, 1:40:00 PM] Loaded plugin: homebridge-config-ui-x
[2/7/2020, 1:40:00 PM] Registering platform 'homebridge-config-ui-x.config'
[2/7/2020, 1:40:00 PM] ---
[2/7/2020, 1:40:00 PM] Loading 2 platforms...
[2/7/2020, 1:40:00 PM] [Config] Initializing config platform...
[2/7/2020, 1:40:00 PM] [Config] Running in Standalone Mode.
[2/7/2020, 1:40:00 PM] [Domoticz] Initializing eDomoticz platform...
[2/7/2020, 1:40:00 PM] Loading 0 accessories...
HAP Warning: Characteristic undefined not in required or optional characteristics for service 24a8d81a-3d7b-4965-9086-7edc3ef4fab3. Adding anyway.
^ For: Power
HAP Warning: Characteristic undefined not in required or optional characteristics for service 24a8d81a-3d7b-4965-9086-7edc3ef4fab3. Adding anyway.
^ For: Power
HAP Warning: Characteristic undefined not in required or optional characteristics for service 24a8d81a-3d7b-4965-9086-7edc3ef4fab3. Adding anyway.
^ For: Power
HAP Warning: Characteristic undefined not in required or optional characteristics for service 24a8d81a-3d7b-4965-9086-7edc3ef4fab3. Adding anyway.
^ For: Rainfall next Hour
HAP Warning: Characteristic undefined not in required or optional characteristics for service 24a8d81a-3d7b-4965-9086-7edc3ef4fab3. Adding anyway.
^ For: Voltage L1
HAP Warning: Characteristic undefined not in required or optional characteristics for service c737b8a5-62fc-4047-b92c-0345bc928445. Adding anyway.
^ For: Gas
HAP Warning: Characteristic undefined not in required or optional characteristics for service c737b8a5-62fc-4047-b92c-0345bc928445. Adding anyway.
^ For: Gas
[2/7/2020, 1:40:00 PM] Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: 8033f7f5-ec19-4167-ac13-e0bafd1bd51d
    at Bridge.Accessory.addBridgedAccessory (/usr/local/lib/node_modules/homebridge/node_modules/hap-nodejs/lib/Accessory.js:268:13)
    at Server._configCachedPlatformAccessories (/usr/local/lib/node_modules/homebridge/lib/server.js:396:18)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:93:8)
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:59:10)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
[2/7/2020, 1:40:00 PM] Got SIGTERM, shutting down Homebridge...
```
Show the Homebridge / Homebridge Config UI X logs here.
```

**Homebridge Config:**

```json
Show your homebridge config.json here
```

**Screenshots:**

**Environment:**

* **Node.js Version**:  v12.15.0 
* **NPM Version**:  v6.13.4 
* **Homebridge Version**: 0.4.50
* **Homebridge Config UI X Version**: v4.10.0 
* **Operating System**: Docker
* **Process Supervisor**: Docker

**Comment by oznu** [comment]
> [2/7/2020, 1:40:00 PM] Error: Cannot add a bridged Accessory with the same UUID as another bridged Accessory: 8033f7f5-ec19-4167-ac13-e0bafd1bd51d

This is an issue with one of your plugins, not Homebridge Config UI X. 

Deleting the `/homebridge/accessories` directory and restating Homebridge usually fixes the problem temporarily (you should raise an issue on the offending plugins project page).

**Comment by raymondsteen** [comment]
That is correct and not the problem why homebridge is not starting at all. This error has been there and is known, it is one accessory. This has been in my log for ages, not impacting the install.
I've removed the directory and restarted: No change. Removed the persist directory and restarted, homebridge is working again.

This happened after updating Config UI X and E-dmomticz plugin.

Working now, but still strange
